src/SFA.DAS.DownloadService.Api/nlog.config

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      autoReload="true"
+      internalLogLevel="info"
+      internalLogFile="internal-nlog.txt"
+      throwExceptions="false">
+
+  <extensions>
+    <add assembly="SFA.DAS.NLog.Targets.Redis" />
+    <add assembly="Microsoft.ApplicationInsights.NLogTarget" />
+  </extensions>
+
+  <!-- the targets to write to -->
+  <targets>
+    <target type="ApplicationInsightsTarget" name="aiTarget" />
+
+    <!-- write to the void aka just remove -->
+    <target xsi:type="Null" name="blackhole" />
+
+    <target xsi:type="Redis" name="redis" connectionStringName="Redis" environmentKeyName="EnvironmentName"
+            appName="das-download-service" includeAllProperties="true" layout="${message}">
```

Comment:
The Api should use a different appName for logs

Reply:
I'll update to das-download-service-api and push right now